### ISSUE TITLE
SD 2.1

### ISSUE THREAD

**Issue by reynoldscem** [KokeCacao/mvdream-hf#1]
Hi, did you get this working for stablediffusion 2.1? For me, it appears to crash on that version, but work for 1.5.

**Comment by KokeCacao** [comment]
Yah... the repo is mostly there for my personal use so I didn't bother to clean up the code. I'll do some cleanup today and make it work for 1.5.

**Comment by KokeCacao** [comment]
Thank you for your feedback. Should be fixed in the latest commit (1aa61db95bf3c369d42f043492ace7eeaffed6c2). You can download v1.5 [here](https://huggingface.co/KokeCacao/mvdream-hf) and v2.1 [here](https://huggingface.co/KokeCacao/mvdream-base-hf). Let me know if they work.

**Comment by reynoldscem** [comment]
Absolutely stunning work! Thank you. I will give them a go and report back!

**Comment by reynoldscem** [comment]
@KokeCacao The exporting seems to work fine! I am unsure however, how to reload the trained model. I have tried 

```
from pipeline_mvdream import MVDreamStableDiffusionPipeline
MVDreamStableDiffusionPipeline.from_pretrained('../converted_sd2/')
```

with and without `use_safetensors`, I get:

`TypeError: MultiViewUNetModel.__init__() missing 6 required positional arguments: 'image_size', 'in_channels', 'model_channels', 'out_channels', 'num_res_blocks', and 'attention_resolutions'`

do you have advice / suggestions?

**Comment by KokeCacao** [comment]
> @KokeCacao The exporting seems to work fine! I am unsure however, how to reload the trained model. I have tried
> 
> ```
> from pipeline_mvdream import MVDreamStableDiffusionPipeline
> MVDreamStableDiffusionPipeline.from_pretrained('../converted_sd2/')
> ```
> 
> with and without `use_safetensors`, I get:
> 
> `TypeError: MultiViewUNetModel.__init__() missing 6 required positional arguments: 'image_size', 'in_channels', 'model_channels', 'out_channels', 'num_res_blocks', and 'attention_resolutions'`
> 
> do you have advice / suggestions?

Thanks for reporting. Sorry for the inconvenience. I didn't test the `pipe` because I was used to using individual components within the pipeline. Should be fixed in 87e9bd7.

**Comment by reynoldscem** [comment]
Absolutely amazing work - thank you!
